.k8s/Makefile

```diff
@@ -0,0 +1,96 @@
+SHELL = /bin/bash
```

Comment:
I might end up to delete this file in favour of using the `.k8s/cli.py` so it can be used for preparing the k8s context.